osf/models/registrations.py

```diff
@@ -201,6 +201,11 @@ def registration_schema(self):
             return self.registered_schema.first()
         return None
 
+    @property
+    def schema_responses(self):
```

Comment:
We already have the `schema_responses` GenericRelationship inherited from `Node`. It lives on Node because `TypedModel` mysteriously blows up if it's on `Registration`.